packages/features/ee/teams/components/MemberList.tsx

```diff
@@ -377,6 +378,24 @@ export default function MemberList(props: Props) {
           return filterValue.includes(rows.getValue(id));
         },
       },
+      {
+        id: "lastLogin",
+        accessorFn: (data) => data.lastLogin,
+        header: "Last Login",
+        cell: ({ row }) => {
+          const { lastLogin } = row.original;
+          return (
+            <div>
+              {lastLogin
+                ? formatLocalizedDateTime(lastLogin, {
+                    dateStyle: "full",
+                    timeStyle: "long",
+                  })
+                : "N/A"}
+            </div>
```

Comment:
I'm not satisfied with the current datetime format in the table. I believe it would be better to present it in this format:- 18/10/2024, 5:06 PM

Reply:
I agree you on this, i didn't like the format either. I have changed it to as you suggested, thanks!